server/controllers/users/getUserByInterest.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
```suggestion
```

please don't disable the ESlint on any file

Reply:
ahmad told us this one is fine 
they did it every where bc we didnt use next yet, when we implementation the function we will remove them dont worry :)

Reply:
anyways don't disable the eslint i know that we will use the next so we don't have an issue if the eslint told us that there is an error on it.
so the eslint error about `next` word does not make sense to me in this PR 
Are you get what i want to say !? @Amoodaa

Reply:
done